scipy/stats/_continuous_distns.py

```diff
@@ -5392,8 +5392,165 @@ def _updated_ctor_param(self):
         return dct
 
 
+def _get_shape_parameters(distribution):
+    """
+    Returns all shape parameters of a scipy distribution
+    including loc and scale
+    @param distribution scipy.stats.rv_continuous
```

Comment:
I will change this (probably next weekend, I'm currently on vacation). Thanks for the feedback.